recipes/unpaper/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+aclocal
+automake --add-missing
+autoconf
+
+./configure --prefix=$PREFIX
+make
+make install
```

Comment:
Is there a make check?

Reply:
Yes, but there are some issues: https://github.com/unpaper/unpaper/issues/79

I've included some but not all tests for now.

Reply:
I see this fails on osx -- I could skip it, or use a very simple test like they do in the homebrew formula: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/unpaper.rb#L37

Thoughts?

Reply:
Sure. As log as we document what we are skipping it should be OK.

Reply:
OK I added a simple test on osx and linked to a relevant issue in the upstream github repository.